src/components/BookCard/BookCard.jsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
 import './BookCard.css';
 
-export default function BookCard({
-	coverImg,
-	authors,
-	title,
-	publishDate,
-}) {
+export default function BookCard({booksData}) {
```

Comment:
Nitpick: This isn't a BookCard, this is a grid of cards. We should make the name of this component reflect what it actually is.